VultisigApp/VultisigApp/macOS/View/PeerDiscoveryView+macOS.swift

```diff
@@ -62,12 +62,9 @@ extension PeerDiscoveryView {
                 .padding()
                 .background(Color.blue600)
                 .cornerRadius(15)
-                .overlay (
-                    RoundedRectangle(cornerRadius: 15)
-                        .strokeBorder(Color.turquoise600, style: StrokeStyle(lineWidth: 2, dash: [58]))
-                )
-                .padding(1)
                 .aspectRatio(contentMode: .fit)
+            
+            outline
```

Comment:
<img width="1711" alt="image" src="https://github.com/user-attachments/assets/14681d8b-cc93-416c-869e-92f50b0ce58e">
the outline doesn't scale with the screen